shared/src/business/useCases/getConsolidatedCasesByCaseInteractor.ts

```diff
@@ -12,12 +14,51 @@ export const getConsolidatedCasesByCaseInteractor = async (
   applicationContext: IApplicationContext,
   { docketNumber }: { docketNumber: string },
 ) => {
+  const user = applicationContext.getCurrentUser();
+
   const consolidatedCases = await applicationContext
     .getPersistenceGateway()
     .getCasesByLeadDocketNumber({
       applicationContext,
       leadDocketNumber: docketNumber,
     });
 
-  return Case.validateRawCollection(consolidatedCases, { applicationContext });
+  if (
+    ![
+      ROLES.petitioner,
+      ROLES.privatePractitioner,
+      ROLES.irsPractitioner,
+    ].includes(user.role)
```

Comment:
This might be easier to maintain and read:

```User.isInternalUser(user.role) || ROLES.irsSuperuser === user.role```

Reply:
Good catch. We have implemented the change.